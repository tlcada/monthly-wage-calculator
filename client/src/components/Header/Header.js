import React from 'react'

import { Link } from 'react-router-dom'
import { Navbar } from 'react-bootstrap'

const Header = () => (
  <Navbar>
    <Navbar.Header>
      <Navbar.Brand>
        <Link to='/'>Monthly Wage Calculator</Link>
      </Navbar.Brand>
      <Navbar.Toggle />
    </Navbar.Header>
  </Navbar>
)

export default Header
