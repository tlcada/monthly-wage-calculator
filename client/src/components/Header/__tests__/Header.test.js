import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from '../Header'

test('Header component should render as expected', () => {
  const component = shallow(<Header />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
