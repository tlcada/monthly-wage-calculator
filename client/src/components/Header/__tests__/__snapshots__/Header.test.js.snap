// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render as expected 1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
>
  <NavbarHeader>
    <NavbarBrand>
      <Link
        replace={false}
        to="/"
      >
        Monthly Wage Calculator
      </Link>
    </NavbarBrand>
    <NavbarToggle />
  </NavbarHeader>
</Uncontrolled(Navbar)>
`;
